impeller/tools/malioc_diff.py

```diff
@@ -285,14 +285,20 @@ def main(argv):
       if compare_shaders(args.after, before_shader, after_shader):
         changed = True
 
+  new_shader = False
   for filename, shaders in after_json.items():
     if filename not in before_json:
-      print(
-          'Shader {} is new. Run with --update to update checked-in results'
-          .format(filename)
-      )
+      new_shader = True
+      print('Shader {} is new.'.format(filename))
       changed = True
 
+  if new_shader:
+    print(
+        'There are new shaders. The golden file must be updated with:\n\n'
+        '$ ./flutter/impeller/tools/malioc_diff.py --before {} --after {} --update'
```

Comment:
I believe there is a precondition that `android_debug_unopt` is built with the `--malioc-path` path for this script to work which is worth mentioning.

Reply:
Or maybe it's better to put that into the script itself.

Reply:
Good call. Updated.